### ISSUE TITLE
[BUG/ISSUE]

### ISSUE THREAD

**Issue by TylerElgiar** [geoschem/geos-chem#1357]
Utah State University

### Description of the problem
Describe your problem here.  Describe the steps to reproduce the problem here, if possible.

I am new to GEOS-Chem and am running a 025x03125 simulation with only mercury chemistry. I was able to run this simulation successfully once. However, I needed to go back and run the simulation on a specific date since the model stopped running in the middle of the run. I am trying to run the model for 2-3 days prior to my desired date, and at least 1 day after my stop date, everything else in the model remains the same as the previous successful run, and I receive these error messages immediately after starting the run:
```console
GEOS-Chem ERROR: No output scheduled on last day of run!
 -> at Is_Last_Day_Good (in module GeosCore/input_mod.F90)
===============================================================================

===============================================================================
GEOS-Chem ERROR: Error reading the "input.geos" file!
 -> at Read_Output_Menu (in module GeosCore/input_mod.F90)
===============================================================================

===============================================================================
GEOS-Chem ERROR: Error in "Read_Output_Menu"!
 -> at Read_Input_File (in module GeosCore/input_mod.F90)
===============================================================================

===============================================================================
GEOS-CHEM ERROR: Error encountered in "Read_Input_File"!
STOP at  -> at GEOS-Chem (in GeosCore/main.F90)
```
### Description of troubleshooting performed
Describe any troubleshooting that you have already performed here. Also include any leads or suspicions here.

I've been in close contact with a GEOS-Chem modeler in our group at Utah State University and they mentioned to  contact github support. I have also taken a close look at the 'input.geos' file to make sure there is output scheduled on the first and last day of the run.

GEOS-Chem version: 12.8.0

### Description of modifications
Describe your modifications here.

None that I'm aware of.

### Log files
- Build log (if applicable):
- Run logs (if applicable):
- Run script (if applicable):

### Software versions
- CMake version:
- Compilers (Intel or GNU, and version):
- NetCDF version:

**Comment by yantosca** [comment]
Thanks for writing @TylerElgiar.  We have information about this error on the GEOS-Chem wiki here: http://wiki.seas.harvard.edu/geos-chem/index.php/Run-time_crashes_and_abnormal_exits#No_output_scheduled_on_last_day_of_run

Basically in the OUTPUT MENU section of input.geos, you need to put a "3" on the last day of the run.

We may try to bring some of the content about GEOS-Chem errors on the wiki over to geos-chem.readthedocs.io at some point, which will be the new user manual in GEOS-Chem 14.0.0 and later.  But for now it's still on the wiki as part of our [Guide to GEOS-Chem error messages](http://wiki.seas.harvard.edu/geos-chem/index.php/Guide_to_GEOS-Chem_error_messages).

**Comment by TylerElgiar** [comment]
Thank you for your response. When I look at the 'OUTPUT MENU' section in 'input.geos' there are 3's all the way down in the first column. Do I need to have a 3 ONLY on the last day of the run?

Thanks,
Tyler
     On Wednesday, August 24, 2022, 11:56:10 AM MDT, Bob Yantosca ***@***.***> wrote:  
 
 


Thanks for writing @TylerElgiar. We have information about this error on the GEOS-Chem wiki here: http://wiki.seas.harvard.edu/geos-chem/index.php/Run-time_crashes_and_abnormal_exits#No_output_scheduled_on_last_day_of_run

Basically in the OUTPUT MENU section of input.geos, you need to put a "3" on the last day of the run.

We may try to bring some of the content about GEOS-Chem errors on the wiki over to geos-chem.readthedocs.io at some point, which will be the new user manual in GEOS-Chem 14.0.0 and later. But for now it's still on the wiki as part of our Guide to GEOS-Chem error messages.

**Comment by yantosca** [comment]
We usually have 3's in the first column because that is the default for monthly mean output.  But if you are using shorter intervals you need to have a 3 on the last day of each run stage.

Most of the bpch diagnostics have been removed in the newer versions of the code (GC 13 and later).  In GC 14 there will be the new Hg simulation following Viral Shah et al 2021 with the mechanism implemented via KPP.   But for the GC 12 versions you still need to set the OUTPUT MENU correctly or you'll get that error.

**Comment by TylerElgiar** [comment]
Okay, I see now. I implemented the change, and it appears to be running. I will contact you again if I encounter another issue similar to this. Thank you for your help!

-Tyler
     On Wednesday, August 24, 2022, 12:15:46 PM MDT, Bob Yantosca ***@***.***> wrote:  
 
 


We usually have 3's in the first column because that is the default for monthly mean output. But if you are using shorter intervals you need to have a 3 on the last day of each run stage.

Most of the bpch diagnostics have been removed in the newer versions of the code (GC 13 and later). In GC 14 there will be the new Hg simulation following Viral Shah et al 2021 with the mechanism implemented via KPP. But for the GC 12 versions you still need to set the OUTPUT MENU correctly or you'll get that error.

**Comment by yantosca** [comment]
Great, I'll close out this issue now.  Glad it works!
